src/components/web3/Web3CTA.tsx

```diff
@@ -21,7 +22,11 @@ export const Web3CTA: React.FC<Props> = ({ onClick }) => {
           avatar, assign moderator privileges using POAPs, and more.
         </p>
         <Button hollow onClick={onClick}>
-          Start a Web3 Call
+          {isSubscribed ? (
+            <>Start a Web3 Call</>
+          ) : (
+            <>Start a Free Trial To Start a Web3 Call</>
+          )}
```

Comment:
Shouldn't this also cause the button to navigate to the account.brave.com if you're not subscribed? Although the message has changed, it still works.

Reply:
Oh just seen @mrose17 said exactly the same thing, apologies.